tests/lib/prepare-restore.sh

```diff
@@ -759,13 +759,15 @@ restore_project_each() {
     # will most likely not function correctly anymore. It looks like this
     # happens with: https://forum.snapcraft.io/t/4101 and is a source of
     # failure in the autopkgtest environment.
-    if dmesg|grep "oom-killer"; then
-        echo "oom-killer got invoked during the tests, this should not happen."
-        echo "Dmesg debug output:"
-        dmesg
-        echo "Meminfo debug output:"
-        cat /proc/meminfo
-        exit 1
+    if [ "$SKIP_OOM_CHECK" != "true" ]; then
+        if dmesg|grep "oom-killer"; then
```

Comment:
It's probably enough to have tests set it to a non empty value
```suggestion
    if [  -n "${SKIP_OOM_CHECK-}" ]; then
        if dmesg|grep "oom-killer"; then
```